### ISSUE TITLE
phpunit should be a dependency

### ISSUE THREAD

**Issue by pmgouveia** [alnutile/pickle#10]
## Detailed description

`sh: vendor/bin/phpunit: No such file or directory
Error with tests....`

## Possible implementation

`$ composer require --dev phpunit/phpunit` solves it

**Comment by drbyte** [comment]
... but it already is a dependency:

https://github.com/alnutile/pickle/blob/master/composer.json#L36

**Comment by pmgouveia** [comment]
yeah but its dev... if its not installed on your system pickel fails to run.
i only installed picke with:
`$ composer require --dev alnutile/pickle`
